.github/workflows/conbench_upload.yml

```diff
@@ -149,3 +155,14 @@ jobs:
             description: description,
             context: 'Benchmark Result Upload'
           })
+
+    - name: Create a GitHub Check benchmark report on the contender comment, and if merge-commit, a comment on the merged PR
+      env:
+        CONBENCH_URL: "https://velox-conbench.voltrondata.run/"
+        GITHUB_APP_ID: "${{ secrets.GH_APP_ID }}"
+        GITHUB_APP_PRIVATE_KEY: "${{ secrets.GH_APP_PRIVATE_KEY }}"
```

Comment:
We just have to make sure these two secrets are populated for this repo before merging.

Reply:
@assignUser brought up that we could use the GitHub Actions App instead, to post the check and PR comment. The advantage would be that it's always using ephemeral credentials, not stored as secrets in the repo. The disadvantage would be that all benchmark-related comments and checks would come from the `@github-actions` bot instead of the `@conbench-facebook` bot (which would be better branded and might be less confusing).

Reply:
This would just require adding `check: write` and `pull-request: write`  to the `permissions` key and maybe some changes on the benchalert side?

Reply:
Yes, I think the only (very small) change on the benchalerts side is https://github.com/conbench/conbench/issues/986

Reply:
I remember we got the conbench github app installed some time ago. Is there anything else that needs to be done to allow you to use it?

Reply:
If the app is installed already, you'll just have to add the `GH_APP_ID` and `GH_APP_PRIVATE_KEY` (contents of the pk file) secrets to this repo's GitHub Actions.